### PR TITLE
Table Layout correction #547

### PR DESCRIPTION
**Pull request by j0rt3g4**
Table Layout Correction #547

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

**Comment by mypil** [comment]
@j0rt3g4 - The table for the "External DNS records required for Skype for Business Online" section is still not fixed. Please check and do the necessary changes to that table. Thank you.

File: Enterprise/external-domain-name-system-records.md [hunk 1]
```diff
@@ -62,12 +62,12 @@ Email customers who are using Exchange Federation will also need the additional
...
+|**MX** <br/> **(Exchange Online)** <br/> |Sends incoming mail for your domain to the Exchange Online service in Office 365.  <br/> [!NOTE] Once email is flowing to Exchange Online, you should remove the MX records that are pointing to your old system.   |**Domain:** For example, contoso.com  <br/> **Target email server:**\<MX token\>.mail.protection.outlook.com  <br/> **Preference/Priority:** lower than any other MX records (this ensures mail is delivered to Exchange Online) - for example 1 or 'low'  <br/>  Find your \<MX token\> by following these steps:  <br/>  Sign in to Office 365, go to Office 365 admin \> Domains.  <br/>  In the Action column for your domain, choose Fix issues.  <br/>  In the MX records section, choose What do I fix?  <br/>  Follow the directions on this page to update your MX record.  <br/> [What is MX priority?](https://go.microsoft.com/fwlink/p/?LinkId=396471) <br/> |
```
Comment (JohanFreelancer9) [inline Enterprise/external-domain-name-system-records.md]:
```suggestion
|**MX** <br/> **(Exchange Online)** <br/> |Sends incoming mail for your domain to the Exchange Online service in Office 365.  <br/> [!NOTE] Once email is flowing to Exchange Online, you should remove the MX records that are pointing to your old system.   |**Domain:** For example, contoso.com  <br/> **Target email server:**\<MX token\>.mail.protection.outlook.com  <br/> **Preference/Priority:** Lower than any other MX records (this ensures mail is delivered to Exchange Online) - for example 1 or 'low'  <br/>  Find your \<MX token\> by following these steps:  <br/>  Sign in to Office 365, go to Office 365 admin \> Domains.  <br/>  In the Action column for your domain, choose Fix issues.  <br/>  In the MX records section, choose What do I fix?  <br/>  Follow the directions on this page to update your MX record.  <br/> [What is MX priority?](https://go.microsoft.com/fwlink/p/?LinkId=396471) <br/> |
```

**Comment by j0rt3g4** [comment]
@mypil I think is done, please check it out again

**Comment by mypil** [comment]
@kelleyvice-msft -  PR has been copy edited and is ready for final review. This is based on issue #547

Thank you.

cc: @kenwith

**Pull request merged**